docs/modules/ROOT/pages/documentation/ecosystem/intellij-plugin/README.adoc

```diff
@@ -1,7 +1,11 @@
 = Payara IntelliJ IDEA Plugin
 
-Payara IntelliJ IDEA tools provide integration of the Payara Platform into IntelliJ IDEA.
-The extension is available in the https://plugins.jetbrains.com/plugin/15114-payara-tools[JetBrains Plugins Marketplace].
+Payara Platform tools provide integration of the Payara Platform into IntelliJ 
+IDEA Ultimate Edition. The extension is available in the https://plugins.jetbrains.com/plugin/15114-payara-platform-tools[JetBrains Plugins Marketplace].
+
+If you are developing the application on the free version of IntelliJ IDEA which 
+is Community Edition then install the Payara Micro Community tools which is also 
```

Comment:
I would reword this to; 

"... If you are developing a application on IntelliJ IDEA Community Edition, then install install Payara Micro Community Tools ..."